drivers/scheduler/oke/oke.go

```diff
@@ -0,0 +1,585 @@
+}
+
+func (o *oke) addUpgradedNodePool(nodePool NodePool, name, version string) error {
+	nodeImageId, err := o.getNodeImageIdToUpgrade(version)
+	if err != nil {
+		return fmt.Errorf("error in getting node image ID %v", err)
+	}
+
+	log.Infof("Adding node-pool with %s with node count [%d]", name, nodePool.NodeConfigDetails.Size)
+	placementConfig := map[string]string{}
+	placementConfig["availability-domain"] = nodePool.NodeConfigDetails.PlacementConfigs[0].AvailabilityDomain
+	placementConfig["subnet-id"] = nodePool.NodeConfigDetails.PlacementConfigs[0].SubnetID
+	placementJson, err := json.Marshal(placementConfig)
+
+	nodeShapeConfig := map[string]float32{}
+	nodeShapeConfig["memory-in-gbs"] = float32(nodePool.NodeShapeConfig.MemoryInGBs)
+
+	nodeShapeConfig["ocpus"] = float32(nodePool.NodeShapeConfig.Ocpus)
+	nodeshapeJson, err := json.Marshal(nodeShapeConfig)
+
+	var cmd string
+	cmd = fmt.Sprintf("%s ce node-pool create --cluster-id %s --compartment-id %s --kubernetes-version %s --name %s --node-shape %s --placement-configs '[%v]' --node-shape-config '%v' --size %d --node-image-id %s",
+		ociCLI, o.clusterId, o.compartmentId, version, name, nodePool.NodeShape, string(placementJson), string(nodeshapeJson), nodePool.NodeConfigDetails.Size, nodeImageId)
+
+	out, stderr, err := osutils.ExecShell(cmd)
+	if err != nil {
+		return fmt.Errorf("error while create node-pool %s, error %v %v", name, err, stderr)
+	}
+	var addNodePoolResp WorkRequest
+	err = json.Unmarshal([]byte(out), &addNodePoolResp)
+	if err != nil {
+		return fmt.Errorf("error while parsing nodepool create output, error %v %v", err, stderr)
+	}
+	nodePoolId, err := o.waitForWorkRequest(addNodePoolResp.OpcWorkRequestId, fmt.Sprintf("Node Pool Add - %s", name))
+	if err != nil {
+		return fmt.Errorf("error in waiting for work request to be over, %v", err)
+	}
+	log.Infof("node-pool %s created successfully, id %s", name, nodePoolId)
+	return nil
+}
+
+func (o *oke) deleteNodePool(nodePool NodePool) error {
+
+	cmd := fmt.Sprintf("%s ce node-pool delete --node-pool-id %s --force", ociCLI, nodePool.ID)
+	out, stderr, err := osutils.ExecShell(cmd)
+	if err != nil {
+		return fmt.Errorf("error while deleting node-pool %s, error %v %v", nodePool.Name, err, stderr)
+	}
+	var nodePoolDeleteResp WorkRequest
+	err = json.Unmarshal([]byte(out), &nodePoolDeleteResp)
+	if err != nil {
+		return fmt.Errorf("error while parsing nodepool delete output, error %v %v", err, stderr)
+	}
+	_, err = o.waitForWorkRequest(nodePoolDeleteResp.OpcWorkRequestId, fmt.Sprintf("Node Pool delete - %s", nodePool.Name))
+	if err != nil {
+		return fmt.Errorf("error in waiting for work request to be over, %v", err)
+	}
+
+	return nil
+
+}
+
+// DeleteNode deletes the given node
+func (o *oke) DeleteNode(node node.Node, timeout time.Duration) error {
+	log.Infof("[Torpedo] Deleting node [%s]", node.Hostname)
```

Comment:
We don't need to mention Torpedo here ^^

```suggestion
	log.Infof("Deleting node [%s]", node.Hostname)
```

Reply:
fixed